src/documentation/reference/ReferenceNode.tsx

```diff
@@ -75,6 +99,41 @@ const ReferenceNode = ({
   );
   const hasDetail = hasDocStringDetail || hasSignatureDetail;
 
+  const handleDragStart = useCallback(
+    (event: React.DragEvent) => {
+      dndDebug("dragstart");
+      event.dataTransfer.dropEffect = "copy";
+
+      const parts = fullName.split(".");
+      const isMicrobit = parts[0] === "microbit";
+      let use = isMicrobit ? parts.slice(1) : parts;
+      use = use.map((p) => classToInstanceMap[p] ?? p);
+      const requiredImport = isMicrobit
+        ? { module: "microbit", name: "*" }
+        : { module: parts[0] };
+      const codeWithImports: CodeWithImports = {
+        code: use.join(".") + (kind === "function" ? "()" : ""),
+        requiredImport,
+      };
```

Comment:
Perhaps the "required import" and "code with imports" bits are a hang over from how this used to be on the old branch? At that point I don't think the drop handling code understood imports and we put more structured data on the clipboard. 

Feels like this could be simpler.

Reply:
We simplified this on liveshare.